src/scripts/main.js

```diff
@@ -1,3 +1,31 @@
 'use strict';
 
 // write code here
+function sortList() {
+  const list = [...document.querySelectorAll('li')];
+
+  list.sort((a, b) => {
+    const item1 = parseInt(a.getAttribute('data-salary').replace(/[\D]/g, ''));
+    const item2 = parseInt(b.getAttribute('data-salary').replace(/[\D]/g, ''));
```

Comment:
You are using `getAttribute` method and then converting the string to a number. You can simplify this by using `dataset` property which allows you to directly access data attributes. For example, you can use `a.dataset.salary` instead of `a.getAttribute('data-salary')`.